.github/workflows/apt.yml

```diff
@@ -0,0 +1,28 @@
+name: Apt Installation
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  push:
+    branches: [ main ]
+    paths:
+      - 'deb.asc'
+      - '.github/workflows/apt.yml'
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'deb.asc'
+      - '.github/workflows/apt.yml'
+jobs:
+  apt_installation:
+    runs-on: ubuntu-latest
```

Comment:
maybe we should (I dont know which version does github uses) verify on the latest stable version of ubuntu ?

Reply:
thought about that, but since I'm updating the repo manually this would be more an annoyance and not really necessary at the moment, since it would fail on PRs in the time window between Github releases and deb repo updates.

Reply:
ah I think I misunderstood you comment, sorry for that.

According to Github's blog it's Ubuntu 22.04, so stable Ubuntu, see https://github.blog/changelog/2022-11-09-github-actions-ubuntu-latest-workflows-will-use-ubuntu-22-04/ ... So I think we are fine here, but good to know for the future.